### PR TITLE
Language Translation to Igbo

### PR DESCRIPTION
**Pull request by ayyushx**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----
The open-source community in Nigeria has been rapidly expanding, with a remarkable 60% growth in contributors since 2020. This surge reflects the increasing interest and participation in collaborative software development. This will enable them to learn in their own native language, making the learning process smoother and more accessible. By offering resources in familiar languages, it ensures that more individuals can fully engage and benefit from the platform.

### CODE REVIEW

**Comment by 9JD** [comment]
Its good

**Comment by Sahar21170** [comment]
Codmora

**Comment by Sahar21170** [comment]
„Cosmora“❤️

**Review by Sahar21170 — COMMENTED** [review]
“Cosmora”

File: _articles/ib/best-practices.md
Comment (Sahar21170) [inline _articles/ib/best-practices.md:12]: ---
layout: default
---

# Contributing to Open Source Guides

Thanks for checking out the Open Source Guides! We're excited to hear and learn from you. Your experiences will benefit others who read and use these guides.

We've put together the following guidelines to help you figure out where you can best be helpful.

## Table of Contents

0. [Types of contributions we're looking for](#types-of-contributions-were-looking-for)
0. [Ground rules & expectations](#ground-rules--expectations)
0. [How to contribute](#how-to-contribute)
0. [Style guide](#style-guide)
0. [Setting up your environment](#setting-up-your-environment)
0. [Community](#community)

## Types of contributions we're looking for

There are many ways you can directly contribute to the guides (in descending order of need):

* Fix editorial inconsistencies or inaccuracies
* [Translate guides into other languages](docs/translations.md)

Interested in contributing to this Open Source Guide? Read on!

## Ground rules & expectations

Before we get started, here are a few things we expect from you (and that you should expect from others):

* Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and projects, which means we likely have different perspectives on "how open source is done." Try to listen to others rather than convince them that your way is correct.
* Open Source Guides are released with a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). By participating in this project, you agree to abide by its terms.
* Please ensure that your contribution passes all tests if you open a pull request. If there are test failures, you will need to address them before we can merge your contribution.
* When adding content, please consider if it is widely valuable. Please don't add references or links to things you or your employer have created, as others will do so if they appreciate it.

## How to contribute

If you'd like to contribute, start by searching through the [pull requests](https://github.com/github/opensource.guide/pulls) to see whether someone else has raised a similar idea or question.

If you don't see your idea listed, and you think it fits into the goals of this guide, open a pull request.

## Style guide

If you're writing content, see the [style guide](./docs/styleguide.md) to help your prose match the rest of the guides.

## Setting up your environment

This site is powered by [Jekyll](https://jekyllrb.com/). Running it on your local machine requires a working [Ruby](https://www.ruby-lang.org/en/) installation with [Bundler](https://bundler.io/) along with [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).

Once you have that set up:

1. Grant execution permissions to the scripts:

```bash
chmod +x script/bootstrap
chmod +x script/server
```

2. Execute the scripts:

```bash
./script/bootstrap
./script/server
```

…and open <http://localhost:4000> in your web browser.

## Community

Discussions about the Open Source Guides take place on this repository's [Issues](https://github.com/github/opensource.guide/issues) and [Pull Requests](https://github.com/github/opensource.guide/pulls) sections. Anybody is welcome to join these conversations.

Wherever possible, do not take these conversations to private channels, including contacting the maintainers directly. Keeping communication public means everybody can benefit and learn from the conversation.

**Review by Sahar21170 — COMMENTED** [review]
Cosmora

**Comment by Sahar21170** [comment]
😊

File: _articles/ib/best-practices.md
Comment (E-456-sya) [reply _articles/ib/best-practices.md:12]: 垃圾邮件

Comment (E-456-sya) [reply _articles/ib/best-practices.md:12]: 垃圾邮件

**Comment by Sahar21170** [comment]
@cosmorama

**Comment by Sahar21170** [comment]
"Cosmora"

**Comment by Sahar21170** [comment]
@Tesla_CEOElonMUSK

**Comment by Sahar21170** [comment]
@Sahar21170

**Comment by Sahar21170** [comment]
"Cosmora"

**Comment by Sahar21170** [comment]
😄

**Comment by Sahar21170** [comment]
✌

**Comment by E-456-sya** [comment]
@Cosmorama ***@***.***> 于 2024年10月14日周一 01:33写道：

**Comment by Sahar21170** [comment]
@cosmorama

**Comment by E-456-sya** [comment]
Hey what's up?

@Cosmorama ***@***.***> 于 2024年10月14日周一 01:51写道：

**Review by Rider5812 — COMMENTED** [review]
https://github.com/github/opensource.guide/pull/3312#pullrequestreview-2365502387

**Comment by ahpook** [comment]
Locking to stop the spam, I will look at this PR during the upcoming week.

File: _articles/ib/metrics.md
Comment (ahpook) [inline _articles/ib/metrics.md:2]: Hi - these `lang` slugs need to be changed on every page: 
```suggestion
lang: ib
```
And ideally the strings in `description` would also be translated

File: _articles/ib/maintaining-balance-for-open-source-maintainers.md
Comment (ahpook) [inline _articles/ib/maintaining-balance-for-open-source-maintainers.md:2]:
```suggestion
lang: ib
```

File: _articles/ib/maintaining-balance-for-open-source-maintainers.md
Comment (ahpook) [inline _articles/ib/maintaining-balance-for-open-source-maintainers.md:3]:
```suggestion
untranslated: false
```

File: _articles/ib/legal.md
Comment (ahpook) [inline _articles/ib/legal.md:2]:
```suggestion
lang: ib
```

File: _articles/ib/leadership-and-governance.md
Comment (ahpook) [inline _articles/ib/leadership-and-governance.md:2]:
```suggestion
lang: ib
```

File: _articles/ib/index.html
Comment (ahpook) [inline _articles/ib/index.html:3]: This title is from the Portuguese translation

File: _articles/ib/how-to-contribute.md
Comment (ahpook) [inline _articles/ib/how-to-contribute.md:2]:
```suggestion
lang: ib
```

File: _articles/ib/getting-paid.md
Comment (ahpook) [inline _articles/ib/getting-paid.md:2]:
```suggestion
lang: ib
```

File: _articles/ib/building-community.md
Comment (ahpook) [inline _articles/ib/building-community.md:2]:
```suggestion
lang: ib
```

File: _articles/ib/best-practices.md
Comment (ahpook) [inline _articles/ib/best-practices.md:2]:
```suggestion
lang: ib
```

**Review by ahpook — CHANGES_REQUESTED** [review]
Thank you for the contribution. This needs some work before it can be merged.

Do you know of other Ásụ̀sụ́ Ìgbò speakers on GitHub? I would appreciate an additional content review from someone fluent in the language to make sure the idioms are correct.

File: _articles/ib/code-of-conduct.md
Comment (ahpook) [inline _articles/ib/code-of-conduct.md:102]:
```suggestion
Verhaltenskodex abụghị iwu a ga-emegharị emegharị. Ị bụ onye nchịkwa Verhaltenskodex ma nwee ọrụ iji soro iwu ndị Verhaltenskodex kwadoro.
```
"Verhaltenskodex" is a word from the German translation.

File: _articles/ib/metrics.md
Comment (ahpook) [inline _articles/ib/metrics.md:91]: This whole section contains German words left over from the `/de` translation.

File: _articles/ib/best-practices.md
Comment (megleg88) [reply _articles/ib/best-practices.md:12]: > 垃圾邮件

**Comment by ayyushx** [comment]
@ahpook  I have made the required changes as mentioned by you.

**Review by nthhanh526 — COMMENTED** [review]
Text.việt nam

**Comment by ahpook** [comment]
There are dozens of technical errors in here causing CI failures along with the language problems. All of the internal heading links are failures, all of the apostrophes were replaced with "smart" apostrophes - probably from translation software. I'm going to close this as it has already taken a lot of time and would introduce a lot of debt to include. Sorry.

**Comment by E-456-sya** [comment]
退订

Eric Sorenson ***@***.***> 于 2024年11月13日周三 上午1:59写道：

**Review by nthhanh526 — COMMENTED** [review]
Cóc lăn

**Comment by nthhanh526** [comment]
> Text.việt nam

~~```
><<
```~~/

**Pull request closed without merging**
